spec/scenarios/posts_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts index page', type: :feature do
+  before :each do
+    @user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
+    @user2 = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
+    @first_post = Post.create(author: @user, title: 'Hello', text: 'This is my post')
+    5.times do |_i|
+      Comment.create(post: @first_post, author: @user2, text: 'Hi Lilly!')
+    end
+    visit user_posts_path(@user)
+  end
+
+  scenario 'displays User profile Picture' do
+    find('img[alt="Lilly"]')
+  end
```

Comment:
- [ ] Most of your tests are passing except this one. Please fix it 🙏 
![image](https://user-images.githubusercontent.com/46839548/181390002-8004f398-e171-4a08-adb0-22a9f9a7451d.png)

Reply:
@codepantha are you availbale

Reply:
@codepantha I dropped a message earlier but you didn't follow up with it

Reply:
@ismailakinkunmi Let's have a call I'm available now!!

Reply:
@ismailakinkunmi I was unavailable then. Please let me know if you still need my help.

Reply:
![Screenshot (192)](https://user-images.githubusercontent.com/37457094/181487769-226d4f64-b227-44d0-be97-45473de7bb34.png)

Reply:
all tests are passing in my repo, in my opinion test are not 100 percent reliable

Reply:
- Well resolved. All tests pass.